python/paddle/fluid/tests/unittests/test_multiclass_nms_op.py

```diff
@@ -210,7 +215,7 @@ def test_check_output(self):
 class TestMulticlassNMSOpNoOutput(TestMulticlassNMSOp):
     def set_argument(self):
         # Here set 2.0 to test the case there is no outputs.
-        # In practical use, 0.0 < score_threshold < 1.0 
+        # In practical use, 0.0 < score_threshold < 1.0
```

Comment:
nothing change

Reply:
The whitespace in the end of line is stripped